app/forms/teacher_interface/add_another_page_form.rb

```diff
@@ -0,0 +1,8 @@
+class TeacherInterface::AddAnotherPageForm
```

Comment:
```suggestion
class TeacherInterface::AddAnotherUploadForm
```

I suggest this is a better name, since it keeps the naming consistent within the app, rather than having two words (page and upload) for the same thing, and `Upload` is the name of the model.